cmake/linux/compilerconfig.cmake

```diff
@@ -39,6 +39,7 @@ set(
   -Wunused-value
   -Wunused-variable
   -Wvla
+  -Wswitch
```

Comment:
Nit: Since this list seems alphabetized, place this alphabetically (between `-Wparentheses` and `-Wuninitialized`) unless there is a reason it cannot be.

Reply:
Re-ordered.